### DISCUSSION TITLE
绳网加载慢解决办法

### DISCUSSION THREAD

**Discussion by HousiqiGithub** [share121/inter-knot#3293] [其他]
![啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊](https://github.com/user-attachments/assets/74dd3c5b-5fbe-402b-912c-e2975d107e55)

![屏幕截图 2024-08-20 163743](https://github.com/user-attachments/assets/3b412da6-f815-465c-b661-3e717eafa416)
![屏幕截图 2024-08-20 163859](https://github.com/user-attachments/assets/ac847ada-f811-4047-966e-7cb5cfb5fae0)
![屏幕截图 2024-08-20 164812](https://github.com/user-attachments/assets/c6d5b685-7f96-4775-8229-1bcc029d984c)

1.从链接下载steam++
https://steampp.net/

2.以管理员身份运行steam++.exe
3.选择网络加速>平台加速>向下拉勾上Github->一键加速

**Comment by EP-office** [comment]
Mac怎么办<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3d389fd2885b008e46d843c552c02bf3_5073438697399152551.png" alt="开拓者 吃瓜脸" width="75" >

**Reply by share121** [reply]
![image](https://github.com/user-attachments/assets/f52a4513-c473-4d8a-9231-33e62a980867)
有 mac 版

**Reply by EP-office** [reply]
已经下好了，谢谢<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by CrayHuan233** [comment]
本人建议使用的：
![image](https://github.com/user-attachments/assets/3750ad66-8fbf-412c-b9f3-3a88b0f82b85)
